content/releases/k-runtime/release-2.16.0.md

```diff
@@ -0,0 +1,12 @@
+---
+url: "/releases/k-runtime-2.16.0/"
+date: 2023-07-04
+type: release-note
+version: "2.16.0"
+title: "Camel-K Runtime release 2.16.0"
+preview: ""
+changelog: ""
+category: "camel-k-runtime"
+milestone: 13
```

Comment:
Seems to be 9 for camel-k-runtime according to https://github.com/apache/camel-k-runtime/milestone/9?closed=1